src/hotspot/share/runtime/vmStructs.cpp

```diff
@@ -211,8 +211,10 @@
   nonstatic_field(ArrayKlass,                  _dimension,                                    int)                                   \
   volatile_nonstatic_field(ArrayKlass,         _higher_dimension,                             ObjArrayKlass*)                        \
   volatile_nonstatic_field(ArrayKlass,         _lower_dimension,                              ArrayKlass*)                           \
-  nonstatic_field(CompiledICHolder,            _holder_metadata,                              Metadata*)                             \
-  nonstatic_field(CompiledICHolder,            _holder_klass,                                 Klass*)                                \
+  volatile_nonstatic_field(CompiledICData,     _speculated_method,                            Method*)                               \
```

Comment:
Please duplicate these `CompiledICData` declarations in `vmStructs_jvmci.cpp` so that they can be used by Graal to ascertain whether ICStubs are in use (Graal is still supporting multi JDK versions).